src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -56,6 +56,7 @@ public function onKernelResponse(FilterResponseEvent $event)
 
         if ($session instanceof Session ? $session->getUsageIndex() !== end($this->sessionUsageStack) : $session->isStarted()) {
             $event->getResponse()
+                ->setExpires(new \DateTime())
```

Comment:
Is it on purpose, that this even overrides the expires header that has been set before this line?

Reply:
this is about forcing the page to be uncacheable anyway. So yes.